docs/source/pillars/computing_technology.md

```diff
@@ -142,8 +142,8 @@ _The ability of the TRE operator to record and analyse data about the usage of t
 * - Your TRE must record usage data.
   - This may include the number of users, number of projects, the amount of data stored, number of datasets, the number of workspaces, etc.
   - Mandatory
-* - Your TRE should record which datasets are accessed, when and by who.
-  - This helps maintain auditability of how sensitive data has been used
+* - Your TRE should record which datasets are accessed, when and by whom.
```

Comment:
```suggestion
* - Your TRE should record which datasets are accessed, when and by who.
```

@jemrobinson @craddm How strongly do you feel about that?

It isn't necessary to use whom here and it feels a little anachronistic.

Reply:
I think @craddm suggested it and it *is* correct, so I'd probably leave it. If you prefer to change it back I'm not going to complain though.

Reply:
Not incorrect, but I do think for modern English it is unusual and comes across as stuffy and possibly a little pretentious.